sub_scripts/testing_process.sh

```diff
@@ -528,9 +342,9 @@ CHECK_URL () {
 					fi
 
 					# Print the first 20 lines of the page
-					ECHO_FORMAT "Extract of the page:\n" "white"
+					ECHO_FORMAT "Extract of the page:\n" "white" >> "$test_result"
 					echo -en "\e[37m"	# Write in 'light grey'
-					lynx -dump -force_html "$script_dir/url_output" | head --lines 20 | tee --append "$test_result"
+					lynx -dump -force_html "$script_dir/url_output" | head --lines 20 >> "$test_result"
```

Comment:
There is the same commit on two different PR, so copy/pasta:

I'd like to keep it, because in some case there are false negative errors cf. [Installation in a sub path](https://ci-apps-unstable.yunohost.org/ci/job/711)

```
Real url: https://sous.domain.tld/path/
HTTP code: 620
Title of the page: Dotclear - Error
Extract of the page:
Dotclear

Database problem

   Something went wrong while trying to read the database.


Test url: sous.domain.tld/path/
Real url: https://sous.domain.tld/path/
HTTP code: 620
Title of the page: Dotclear - Error
Extract of the page:
Dotclear

Database problem

   Something went wrong while trying to read the database.
```

In a perfect world we want to detect this and remove false negative errors, but we can't handle all specific errors